modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/connectors/jms/utils/JMSConstants.java

```diff
@@ -1,84 +0,0 @@
-/*
```

Comment:
Why we have removed these constants ? I think this made ballerina to directly depend on jms transport